### PR TITLE
update spring boot

### PR DESCRIPTION
**Pull request by amhudson**
Closes #
https://github.com/boomerang-io/roadmap/issues/339

#### Changelog

**New**

- N/A

**Changed**

- Update Spring Boot to 2.6.6

**Removed**

- N/A

#### Testing / Reviewing

Tested and verified locally

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		<jacoco.version>0.8.2</jacoco.version>
 		<spring.data>2.1.9.RELEASE</spring.data>
```
Comment (AndreiOla) [inline pom.xml]: This lib needs to be correlated also with spring-boot 2.6.6: spring-data-commons | 2.6.3

File: pom.xml [hunk 2]
```diff
@@ -91,88 +91,75 @@
...
+		<!-- 	<dependency>
 			<groupId>net.boomerangplatform</groupId>
 			<artifactId>lib-mongo</artifactId>
 			<version>4.0.26</version>
-		</dependency>-->		
+		</dependency>-->
 		<dependency>
 			<groupId>net.boomerangplatform</groupId>
 			<artifactId>lib-util</artifactId>
 			<version>1.0.0</version>
```
Comment (AndreiOla) [inline pom.xml]: I think this needs to be updated to  1.2.4. Not sure where is the repo for these libraries.

File: pom.xml [hunk 3]
```diff
@@ -91,88 +91,75 @@
...
+		<!-- 	<dependency>
 			<groupId>net.boomerangplatform</groupId>
 			<artifactId>lib-security</artifactId>
 			<version>1.0.3</version>
-		</dependency>-->		
+		</dependency>-->
 		<dependency>
 			<groupId>net.boomerangplatform</groupId>
 			<artifactId>lib-rest</artifactId>
 			<version>1.0.1</version>
```
Comment (AndreiOla) [inline pom.xml]: This library seems to be renamed 4 years ago and the last tag is 0.0.7, so I'm not sure where that 1.0.1 version is taken from. Might be a public repository?

**Review by AndreiOla — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (amhudson) [reply pom.xml]: `lib-rest` is now called `lib-proxy`. updated with latest

**Review by AndreiOla — APPROVED** [review]

**Comment by amhudson** [comment]
@tlawrie Did you want to review?

**Review by morarucostel — APPROVED** [review]

**Pull request merged**